test/Config/SessionConfigTest.php

```diff
@@ -479,13 +480,15 @@ public function testUseOnlyCookiesDefaultsToIniSettings(): void
         self::assertSame((bool) ini_get('session.use_only_cookies'), $this->config->getUseOnlyCookies());
     }
 
+    #[IgnoreDeprecations]
```

Comment:
Is `IgnoreDeprections` required because of unresolvable deprecations in `SessionConfig`?

Reply:
Yes, it's the purpose of the method itself that is deprecated in PHP 8.4, so I tag it as deprecated in the laminas api and ignore deprecation in test